app/controllers/csv_files_controller.rb

```diff
@@ -0,0 +1,109 @@
+class CsvFilesController < ApplicationController
+  include Alertable
+
+  # Kluge for brakeman
+  STI = { 
```

Comment:
Alertable just produces an unsigned list of action errors, and is used by every controller.